### PR TITLE
course detail

### PR DESCRIPTION
**Pull request by VuVanDuc**
course_detail html - Initial
course_detail_trainer - Initial

### CODE REVIEW

**Comment by VuVanDuc** [comment]
ready

File: TrainingSystem/course/templates/course/course_detail.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                        </a>
+                        <a href="#">
+                            <button type="button" class="btn btn-primary">View activities</button>
+                        </a>
+                    </div>
+                {% endif %}
+            </div>
+        </form>
+    </div>
+{% endblock %}
```
Comment (huyhm-0966) [inline TrainingSystem/course/templates/course/course_detail.html]: end lien em nè

**Comment by huyhm-0966** [comment]
commented

**Comment by VuVanDuc** [comment]
ready

**Pull request merged**